pkgs/applications/editors/vscode/default.nix

```diff
@@ -49,8 +49,8 @@ in
     };
 
     buildInputs = if stdenv.system == "x86_64-darwin"
-      then [ unzip makeWrapper libXScrnSaver libsecret ]
-      else [ makeWrapper libXScrnSaver libxkbfile libsecret ];
+      then [ unzip makeWrapper wrapGAppsHook libXScrnSaver libsecret ]
```

Comment:
Darwin probably does not need it. If it is like Atom, they use a different front-end for MacOS.